### PR TITLE
Blitz

### PR DESCRIPTION
**Pull request by mattco98**
Sample output: https://hastebin.com/nikufekoqe.json

### CODE REVIEW

File: processors/games/HungerGames.js [hunk 1]
```diff
@@ -1,8 +1,221 @@
...
+    kills,
+    kd: getRatio(kills, deaths),
+    wins,
+    team_wins: wins_teams,
+    win_loss: getRatio(wins + wins_teams, wins + wins_teams + deaths),
+    weekly_kills: getWeeklyStat(weekly_kills_a, weekly_kills_b),
+    monthly_kills: getMonthlyStat(monthly_kills_a, monthly_kills_b),
+    rambo_wins,
+    random_wins,
+    kits_levels: {
```
Comment (builder-247) [inline processors/games/HungerGames.js]: Having values 1-indexed would make the API more accessible for newer developers, and values can be more easily used in ui.

File: processors/games/HungerGames.js [hunk 2]
```diff
@@ -1,8 +1,221 @@
...
-  coins = 0,
-}) => ({
+  // General
   coins,
-});
+  deaths,
+  kills,
+  wins,
+  wins_teams,
+  damage_taken,
```
Comment (builder-247) [inline processors/games/HungerGames.js]: Looks like you forgot to return some values, which causes linting to [fail](https://travis-ci.org/slothpixel/core/builds/438949874?utm_source=github_status&utm_medium=notification).

**Comment by builder-247** [comment]
Part of #20.

File: processors/games/HungerGames.js (hunk 1 above)
Comment (mattco98) [reply processors/games/HungerGames.js]: Ahh I see, the kit levels start at zero. Right, adding one would make sense

**Comment by builder-247** [comment]
Linting 😅

**Pull request merged**